milli/src/update/index_documents/transform.rs

```diff
@@ -269,6 +284,8 @@ impl Transform<'_, '_> {
         let mut uuid_buffer = [0; uuid::adapter::Hyphenated::LENGTH];
         let mut documents_count = 0;
 
+        let mut fields_ids_distribution = HashMap::<FieldId, u64>::default();
```

Comment:
```suggestion
        let mut fields_ids_distribution = HashMap::new();
```

Reply:
It was intended 😄 
```
error[E0282]: type annotations needed for `HashMap<u8, V>`
   --> milli/src/update/index_documents/transform.rs:201:24
    |
140 |         let mut fields_ids_distribution = HashMap::new();
    |             --------------------------- consider giving `fields_ids_distribution` the explicit type `HashMap<u8, V>`, with the type parameters specified
...
201 |         for (field_id, count) in fields_ids_distribution {
    |                        ^^^^^ cannot infer type

```

Reply:
hum, yeah you can only keep this though :)

```rust
let mut fields_ids_distribution = HashMap::<_, u64>::new();
```